com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/GameProjectBuilder.java

```diff
@@ -118,8 +122,10 @@ private RandomAccessFile createRandomAccessFile(File handle) throws IOException
             }
         }
 
-        project.buildResource(input, CopyCustomResourcesBuilder.class);
-
+        // ignore custom resources if processing a project file other than game.project
+        if (!nonStandardGameProjectFile) {
+            project.buildResource(input, CopyCustomResourcesBuilder.class);
+        }
```

Comment:
Is this the only thing we turn off?
I think the only thing we should do here is to write the output file?